src/WebApiContrib.Core.Formatter.MessagePack/MessagePackFormatterOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MessagePack;
+using MessagePack.Resolvers;
+using System.Collections.Generic;
+
+namespace WebApiContrib.Core.Formatter.MessagePack
+{
+    public class MessagePackFormatterOptions
+    {
+        public IFormatterResolver FormatterResolver { get; set; } = ContractlessStandardResolver.Instance;
+
+        public HashSet<string> SupportedContentTypes { get; set; } = new HashSet<string> { "application/x-msgpack" };
```

Comment:
looks like the community settled on `application/x-msgpack` though https://github.com/msgpack/msgpack/issues/194#issuecomment-293842226
That said, it's configurable for this very reason. Or we can just add both out of the box?